requirements/local.txt

```diff
@@ -11,3 +11,8 @@ coverage
 # The version shouldn't really matter here, as it's not part of the running
 # Django project.
 Sphinx
+
+# BeautifulSoup's encoding detection becomes more accurate if chardet or
+# cchardet is installed. chardet is older; cchardet is much faster.
+# Use chardet here since installing cchardet caused trouble on mac.
```

Comment:
Hmm, OK. [This cchardet issue comment](https://github.com/PyYoshi/cChardet/issues/37#issuecomment-569967562) suggests that upgrading to Python 3.7 may fix cchardet installation on Mac. Not our immediate priority I guess, but we'll see when we can upgrade from 3.6 to 3.7, then we can look at cchardet again.

Reply:
sounds good.